task/04-arrays-tasks.js

```diff
@@ -320,9 +400,13 @@ function getPositivesCount(arr) {
  *   [ 'nine','eight','nine','eight'] => [ 'eight','eight','nine','nine']
  *   [ 'one','one','one','zero' ]     => [ 'zero','one','one','one' ]
  */
-function sortDigitNamesByNumericOrder(arr) {
-   throw new Error('Not implemented');
-}
+
+   function sortDigitNamesByNumericOrder(arr) {
+      return arr.sort((a, b) => {
+         var value = {'zero' :0, 'one': 1, 'two': 2, 'three': 3, 'four': 4, 'five': 5, 'six': 6, 'seven': 7, 'eight': 8, 'nine': 9};
```

Comment:
`var` is deprecated. It has no sense to use for browsers code even currently.

Reply:
my bad)